app/v1/gov/metrics.go

```diff
@@ -20,10 +17,12 @@ const (
 	ParamKeyLabel    = "parameter_key"
 )
 
+type Label = stdprometheus.Labels
+
 type Metrics struct {
-	ProposalStatus metrics.Gauge // 0:DepositPeriod 1:VotingPeriod 2:Pass 3:Reject 4:Other
-	Vote           metrics.Gauge // 0:Yes 1:No 2:NoWithVeto 3:Abstain
-	Param          metrics.Gauge
+	ProposalStatus *stdprometheus.GaugeVec // 1:DepositPeriod 2:VotingPeriod 3:Passed 4:Rejected
+	Vote           *stdprometheus.GaugeVec // 0:Yes 1:No 2:NoWithVeto 3:Abstain
```

Comment:
Comment for `Vote` should be consistent with constant definitions in `depositsvotes.go`